examples/with-ethers/.env.local.example

```diff
@@ -3,3 +3,4 @@ API_PRIVATE_KEY="<API Private Key>"
 BASE_URL="https://coordinator-beta.turnkey.io"
 ORGANIZATION_ID="<Organization ID>"
 PRIVATE_KEY_ID="<(Crypto) Private key ID>"
+INFURA_KEY="<Infura API Key>"
```

Comment:
Nit: `node url` or `provider url` might be a better abstraction

Reply:
that's a good call. though for now, since infura provider constructor expects specifically an api key for some reason, i'll stick with this

```
let provider = new ethers.providers.InfuraProvider(DEFAULT_ENV, process.env.INFURA_KEY || DEFAULT_INFURA_COMMUNITY_KEY);
```